fineract-core/src/main/java/org/apache/fineract/infrastructure/event/business/service/BusinessEventNotifierServiceImpl.java

```diff
@@ -100,7 +99,7 @@ public void notifyPostBusinessEvent(BusinessEvent<?> businessEvent) {
         }
         if (isExternalEvent && isExternalEventPostingEnabled()) {
             // we only want to create external events for operations that were successful, hence the post listener
-            if (isExternalEventConfiguredForPosting(businessEvent.getType())) {
+            if (externalBusinessEventConfigurationService.isExternalEventConfiguredForPosting(businessEvent.getType())) {
```

Comment:
You might wanna provide the whole business event, instead of the type only. It provides more options to any alternative implementations.

Reply:
Yeah, that makes sense, updated it.